app/models/medium.rb

```diff
@@ -1125,7 +1125,22 @@ def subscribed_users
 
     Lecture.find_by(id: teachable.lecture_id).user_ids
   end
-
+  
+  def get_annotations_status
+    if annotations_status != 0
+      return annotations_status
+    else
+      return lecture.annotations_status
+    end
+  end
```

Comment:
It would probably be more rubyesque to write this as

```
def get_annotations_status
  return annotations_status if annotations_status != 0
  lecture.annotation_status
end
```